sim/core/racials.go

```diff
@@ -153,36 +152,41 @@ func makeBerserkingCooldown(character *Character, customPercentage float64, time
 	}
 
 	calcBerserkingPct := func() float64 {
-		if customPercentage == 0 {
-			healthPctMissing := 1 - character.CurrentHealthPercent()
-			// 10% base + 1/3 of missing health percentage up to a max of 30%
-			return math.Min(.1+(healthPctMissing/3), .3)
-		} else {
+		if customPercentage != 0 {
 			return customPercentage
 		}
+		// from 10% at full health to 30% at 40% or less health
+		switch hp := character.CurrentHealthPercent(); {
+		case hp >= 1:
+			return 0.1
+		case hp <= 0.4:
+			return 0.3
+		default:
+			return 0.1 + (1-hp)/3
+		}
```

Comment:
❓ Do the numbers change at all here? I'm trying to understand why the shaman tests changed

Reply:
Tests changed because of "added mana/rage/energy costs to Berserking". This here is the "clamp berserkingPct between 0.1 and 0.3 part", for the odd-ball case where Berserking is activated while e.g. Last Stand is active.

Reply:
So the numbers here haven't changed at all, since that latter scenario isn't tested (yet).